### DISCUSSION TITLE
Back after long hiatus - all good, just no log in docker

### DISCUSSION THREAD

**Discussion by cheme75** [mrlt8/docker-wyze-bridge#1212] [General]
Finally booted up the DWB PC at new house - only got the wyze db cam now. Got the wyze api id and key, fixed up yml file, pulled latest 2.9.2 - bingo, all working well. Web ui did ask for login even with the api id and key which only worked with my wyze username and pw - curious if the wb auth stuff replaces the need to use the wyze login? like we can set our own login credentials?

Also, for some reason I am not seeing any log, stats show activity, and running on LAN is fine since I can log into web ui from any PC/device, just odd no log. Don't recall anything to turn log on or off in the past.

**Comment by mrlt8** [comment]
I know it's a little confusing, but some users were running the bridge with ports open, so I've set `WB_AUTH` to be true by default.

If you're running the bridge completely local, then you can set `WB_AUTH=False` to go back to the way it was before.

You can change the web ui login with `WB_USERNAME` and `WB_PASSWORD`.

If WB_USERNAME and WB_PASSWORD are not set, the system will try to use WYZE_EMAIL and WYZE_PASSWORD.

In case neither sets of credentials are provided, the username will default to wbadmin with a randomly generated WB_PASSWORD, which will be logged and stored in a wb_password file within the tokens directory.



More info on the 2.9.0 release page: https://github.com/mrlt8/docker-wyze-bridge/releases/tag/v2.9.0

**Reply by cheme75** [reply]
Ah. I see. Tried to keep up with posts on updates but 8 months or so was a lot to digest. Overall, it's pretty cool things still work. Lots of improvements for the die hard wyze users. Will set wb_auth to false or setup easier to use credentials since I'm not too concerned about security. My wyze login is plenty difficult on its own. Would be nice to view log in docker so will try to figure out why nothing is showing. Update: all is good, not sure why log was slow to appear but its all there now and I can see what's going on which is not much, but good to have log just in case I goof up something else. Frankly, I will probably transition away from wyze. I got no need for using the v2 cams indoors and no good place like old house to place them for looking outside, on the exterior I added extra outlets but few are in good places to keep the v2s under cover so highly doubtful I'll bother to use them. As it happens, blink beta testing left me with 2 of their outdoor battery cams which work fairly well without a subscription, so I may just use those. Will see - If I can get a good deal on the old wyze v3 I may try them since they can handle the exterior better. Anyway, all is good on with my DB cam and docker seems fine even if still at ver 4.12.
